spec/policies/v2/profile_policy_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```

Comment:
@xmicha82 @RoamingNoMaD when creating new files, in 99% of the cases you should write tests for it.